package/src/__tests__/Slider.test.tsx

```diff
@@ -1,6 +1,5 @@
-/* @flow */
-
 import * as React from 'react';
+//@ts-ignore
```

Comment:
Why this ignore? there's https://www.npmjs.com/package/@types/react-test-renderer that we can use